scripts/feature_tests/upgrade/upgrade_vars.sh

```diff
@@ -10,7 +10,7 @@ export CAPI_VERSION="${CAPI_VERSION:-v1alpha4}"
 export CAPI_REL_TO_VERSION="v1.0.1"
```

Comment:
Just a note, we could also update this to the latest version in upgrade tests, but it can be done in a separate patch